src/data/team.js

```diff
@@ -335,7 +335,7 @@ Various design podcasts including [SimpleBits](https://show.simplebits.com/), [H
     cssFeature: 'word-break: break-word',
     questionOne: `Up until now, I've been mostly working Front End Engineering roles, but I've always been interested in finding something that was a good mix between Front End and Design. Design Systems feels like the perfect hybrid between the two! \n \n I'm also really into accessibility, and I found that having a design system that enforces accessible standards is one of the easiest ways to make progress on getting a team on-boarded with accessibility best practices and getting those practices implemented in production \n \n I'm super excited about our React component library that we've been working on over the past few months!`,
     questionTwo: `Some people & projects that have been inspiring to me are: are Nicole Sullivan, Vaidehi Joshi's [BaseCS blog](https://medium.com/basecs) &
-     , Mina Markham's [slay.sh](https://slay.sh/), Una Kravets, Suz Hinton's
+     , Mina Markham's [slay.sh](https://slay.sh/), Una Kravetz, Suz Hinton's
```

Comment:
It seems I guessed wrong. I should have used Google instead of assuming that an anchor would be more likely to be correct:
https://github.com/primer/primer.style/blob/2f4475faa30a89d5d1e1807c132e46a6632711f7/src/data/team.js#L325